tests/integration_tests/functional/test_pause_resume.py

```diff
@@ -141,7 +141,9 @@ def test_kvmclock_ctrl(uvm_plain_any):
     microvm = uvm_plain_any
     microvm.help.enable_console()
     microvm.spawn()
-    microvm.basic_config()
+    # Testing with 1 vcpu due to rare kernel bug which raises soft lockup warning when we have
+    # multiple vcpus
```

Comment:
Is it worth keeping the "soft lockup" assert if we are avoiding it this way? Looking at the history, that assert was used to see if we had an RCU soft lockup, maybe we should be more specific in which kind of soft lockup we are testing for?

Reply:
Probably didn't explain it well, with the KVMCLOCK_CTRL removed we do see the `soft lockup` warning. So it does assert that the call is working. This just saves the issue with it in with multiple vcpus

Reply:
Is there a link for the kernel bug that we can provide? Ie, what makes use believe it's a kernel bug?

Reply:
I agree with Nikita. Jack, what makes you believe that this is a kernel bug? Have you found something relevant in the LKML? If so, please could you update both the PR description, commit message and comment here to point to that and explain what the issue is?

Reply:
I haven't been able to find anything related to this specifically. I am assuming it is, as the ioctl has worked for the rcu_shed self-detected stall warning which doesn't print the warning, it's only the watchdog which throws this.